spec/helpers/rdvs_helper_spec.rb

```diff
@@ -157,4 +159,42 @@
       expect(change_status_confirmation_message(rdv, "revoked")).to eq(expected)
     end
   end
+
+  describe "#rdv_success_flash" do
+    context "when we flash for creation" do
+      context "when the rdv starts in the past" do
+        it "flashes success with an alert about the starting date" do
+          rdv.starts_at = 1.week.ago
+          expect(rdv_success_flash(rdv, what: :create)).to eq({ alert: I18n.t("admin.rdvs.message.success.create.date_alert", distance: distance_of_time_in_words_to_now(rdv.starts_at)) })
```

Comment:
Je trouve plus lisible dans les tests (comme tu l'as fait dans les autres) de directement comparer à la string brut (sans passer par I18n). Préférence perso

Reply:
J'ai vu qu'il y avait les deux approches dans le projet. Je n'ai pas tellement d'avis. Ici, dans la mesure où je me suis embêté à mettre les trads, ça me semblait plus logique de les utiliser :)

Reply:
Du coup le test est plus vraiment end to end. On ne teste pas ici que c'est vraiment le contenu qu'on veut qui s'affiche mais le contenu qui est présent dans les trads. Il suffit que tu aies fait des typos dans les trads par ex, maintenant ou quand on les fait évoluer plus tard pour que ça puisse poser problème. Quand on changera les trads les tests ne changeront pas, alors que le comportement de l'app changera (et j'aimerais le voir dans les tests).

Aucune obligation à changer ça bien sûr mais c'était pour mieux expliquer ma préférence :)

Reply:
J'achète l'argument qui vise à dire : si le texte change, accidentellement ou non, on a envie de le savoir grâce à spec qui échoue. Du coup j'ai changé :)